src/components/ContactOption.jsx

```diff
@@ -13,10 +13,9 @@ export default function ContactOption(prop) {
     prop.onSelectContact(selectedContact);
   }
 
-  // Fetch all contacts when you first load the page
   useEffect(() => {
     window
-      .fetch('/api/v1/contacts/all', {
+      .fetch('/api/v1/contacts', {
```

Comment:
I believe this should be '.fetch('/api/v1/contacts/all'

Reply:
It's good, I unified the contacts API to `/api/v1/contacts`. See #82